### PR TITLE
fixed random time issues with the metrics tests

### PR DESCRIPTION
**Pull request by mauro-fanton**
Fixed an issue with the metric tests. These were running async tests, that may have caused the random time issue which depend of how the test run and the speed. Now blocking on the function cal and then assert the result.

### CODE REVIEW

File: test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/metrics/MetricsUtils1Spec.scala [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import org.mockito.Mockito.{times, verifyNoInteractions, verifyNoMoreInteractions, when}
+import org.scalatest.compatible.Assertion
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class MetricsUtils1Spec extends AsyncFlatSpec {
```
Comment (IoannisLazaridis-oc) [inline test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/metrics/MetricsUtils1Spec.scala:31]: We should  find a better name for this class

Comment (mauro-fanton) [reply test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/metrics/MetricsUtils1Spec.scala:31]: this need to be deleted. Forgot to do that

Comment (mauro-fanton) [reply test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/metrics/MetricsUtils1Spec.scala]: done

**Review by IoannisLazaridis-oc — DISMISSED** [review]
LGTM

**Review by IoannisLazaridis-oc — APPROVED** [review]

**Pull request merged**